coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,5 @@
 class InvalidFilterException(Exception):
-    pass
+    def __init__(self, filter_name, available_filters):
+        super().__init__('{!r} is an invalid filter. Available filters: {}'
+                         .format(filter_name, ', '.join(sorted
```

Comment:
move the second argument to the next line under `filter_name`, so you dont split `sorted(..)` across lines.